app/models/git_repository.rb

```diff
@@ -76,6 +76,12 @@ def valid_url?
     valid
   end
 
+  def downstream_commit?(old_commit, new_commit)
+    cmd = "git rev-list --boundary #{old_commit}..#{new_commit}"
```

Comment:
I think this could be a bit more descriptive?

we should be able to do something like:

`git merge-base --is-ancestor #{old_commit} #{new_commit}`

in which case we just use the value of `success` although it will be the inverse of what `git merge-base` actually returns because of how the TerminalExecutor works. When the old commit is an ancestor the git command returns 1, TerminalExecutor interprets it as an error (because it isn't 0) and sets success to false.

Are you ok with that?